src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useTradeExecution.tsx

```diff
@@ -211,6 +215,13 @@ export const useTradeExecution = (
         if (isLastHop && !hasMixpanelSuccessOrFailFiredRef.current) {
           trackMixpanelEvent(MixPanelEvent.TradeSuccess)
           hasMixpanelSuccessOrFailFiredRef.current = true
+
+          if (
+            bnOrZero(thorVotingPower).toNumber() >= THORSWAP_UNIT_THRESHOLD &&
+            new Date().getUTCFullYear() < THORSWAP_MAXIMUM_YEAR_TRESHOLD
+          ) {
+            trackMixpanelEvent(MixPanelEvent.ThorDiscountTradeSuccess)
```

Comment:
it may be beneficial to track the full state of thor discount trades (confirm, success, and failed)

Reply:
Done in [05daabc](https://github.com/shapeshift/web/pull/8044/commits/05daabc89d2082b788ce937e871349e89d2193ba)